### ISSUE TITLE
train_ema_*.py not in repo

### ISSUE THREAD

**Issue by matthijsvk** [sanyalsunny111/Early_Weight_Avg#1]
Hi, in the README you say

> torchrun --standalone --nproc_per_node=3 train_ema_medium.py

and 

> Please refer to the large and medium configurations of the EMA scripts for running large and medium versions SWA and LAWA.

but I don't see the script `train_ema_medium.py` in the code folder.
Could you please upload this one?

**Comment by sanyalsunny111** [comment]
Thanks for using this repo, all train_small.py, train_medium.py and train_large.py has inbuilt EMA so, it simultaneously performs vanilla training along with EMA. Although the ema checkpoints are not save but the log files saves the val loss for ema models. I will update the Readme.

**Comment by matthijsvk** [comment]
Can the script then also use your method for improving training? Or how
should I apply it?

**Comment by sanyalsunny111** [comment]
Updated Readme. Train and save ckpts then run lawa.py for averaging.

**Comment by sanyalsunny111** [comment]
Hey @matthijsvk did this work?
